pkg/apis/autoscaling/v1alpha1/pa_lifecycle_test.go

```diff
@@ -21,6 +21,7 @@ import (
 
```

Comment:
There are few strings here (`"kpa.autoscaling.knative.dev"` and `"hpa.autoscaling.knative.dev"`) that can be replaced with consts too

Reply:
Modifiied